django/test/utils.py

```diff
@@ -401,18 +403,33 @@ def enable(self):
         self.wrapped = settings._wrapped
         settings._wrapped = override
         for key, new_value in self.options.items():
-            setting_changed.send(sender=settings._wrapped.__class__,
-                                 setting=key, value=new_value, enter=True)
+            try:
+                setting_changed.send(sender=settings._wrapped.__class__,
+                                     setting=key, value=new_value, enter=True)
+            except Exception as ex:
```

Comment:
Use `as e` or as `as exc` to match other code.

Reply:
Not sure what other code you mean, but I'll make the fix.